src/config.js

```diff
@@ -6,33 +6,31 @@ const config = cfg({
   allowedOrigins: {
     env: 'ALLOWED_ORIGINS',
     type: 'string',
-    default: '*',
+    required: true,
   },
   allowedHeaders: {
     env: 'ALLOWED_HEADERS',
     type: 'string',
-    default: '*',
+    required: true,
   },
   appName: {
     env: 'APP_NAME',
     type: 'string',
-    default: 'fusion-node-express-api-skeleton',
+    required: true,
   },
   desiredPort: {
     env: 'PORT',
     type: 'integer',
-    default: '3000',
+    required: true,
   },
   debug: {
     env: 'DEBUG',
     type: 'string',
-    default: 'fusion-node-express-api-skeleton:*',
   },
   nodeEnv: {
```

Comment:
Unless there's a compelling reason to keep it this `nodeEnv` var should be pulled out. Tying app behaviors to an unrelated magic enumerated value is bad practice.

Reply:
This provides a way to filter out log messages not related to the app, so in this case it's okay:

![image](https://user-images.githubusercontent.com/6438381/38525495-9285af60-3c20-11e8-9f11-05aee8dad32c.png)